src/hooks/useClubPruneReport.ts

```
@@ -0,0 +1,86 @@
+    setIsDryRunLoading(true);
+    try {
+      const { data, error: rpcError } = await supabase.rpc(
+        "prune_club_rosters",
+        { p_dry_run: true, p_inactivity_threshold_months: 18 },
+      );
+      if (rpcError) throw rpcError;
+      await fetchReport();
+      return data as PurgeSummary;
+    } catch (err) {
+      const msg =
+        err instanceof Error ? err.message : "Failed to run dry-run purge";
```

ESLint

[error] Delete `⏎·······`

(prettier/prettier)